### PR TITLE
Gseal 74 rewrite story titles

### PR DESCRIPTION
**Pull request by gpseal**
A few small additions to the re-writer process:

- Titles now rewritten and added to article table
- Fictional authors added to stories (randomised each time a story is opened)
- First paragraph of stories now bold

Also updated category title colours inside articles which were not working since we moved to categories being stored in the database

### CODE REVIEW

**Review by glwatkins — APPROVED** [review]
Yes looks good!

**Review by ArtemKech — APPROVED** [review]
Great changes for our website and `re-writer` module. Not sure about the part where we assign fictional authors to the stories. 
Need to figure out how to get actual (legit) names of the authors for the stories. But this is the issue for another sprint. 
Good job overall!

File: database/src/init.sql [hunk 1]
```diff
@@ -26,6 +27,10 @@ create table Article_Raw (
         retrieved_date timestamptz not null default now()
 );
 
+create table Authors (
```
Comment (AardhynLavender) [inline database/src/init.sql:30]: `Author` singular?

File: database/src/init.sql [hunk 2]
```diff
@@ -46,4 +51,12 @@ values  ('news', 'generic news articles', '#cc0099'),
         ('food', 'food, cooking, and recipes articles', '#5039a3'),
         ('sport', 'sport, fitness, and exercise articles', '#39a375');
 
+insert into Authors (name)
+values ('John Doe'),
+        ('Jane Doe'),
+        ('Joe Blow'),
+        ('Joey Sixpack'),
+        ('Tommy Writer'),
+        ('Sarah Jones');
```
Comment (AardhynLavender) [inline database/src/init.sql:60]: Can we get ChatGPT to generate something like 100 believable names instead.

**Review by AardhynLavender — CHANGES_REQUESTED** [review]
Epic. Just a couple of things

File: database/src/init.sql (hunk 2 above)
Comment (AardhynLavender) [reply database/src/init.sql:60]:
```plaintext
generate 100 author names in a sql insert statement
```

```sql
insert Author (name) values
('Eleanor Mitchell'),
('Benjamin Hudson'),
('Olivia Peterson'),
('William Anderson'),
('Sophia Richardson'),
('Daniel Martin'),
('Ava Thompson'),
('James Rodriguez'),
('Emma Foster'),
('Michael Brooks'),
('Isabella Hughes'),
('John Mitchell'),
('Mia Carter'),
('David Cooper'),
('Sophie Walker'),
('Joseph Wright'),
('Amelia White'),
('Matthew King'),
('Emily Scott'),
('Christopher Morgan'),
('Grace Davis'),
('Andrew Lee'),
('Avery Evans'),
('Ryan Turner'),
('Chloe Bennett'),
('Nicholas Murphy'),
('Harper Collins'),
('Ethan Hill'),
('Liam Allen'),
('Aria Phillips'),
('Daniel Young'),
('Layla Hall'),
('Alexander Wright'),
('Charlotte Adams'),
('Logan Green'),
('Madison Cook'),
('Benjamin Gray'),
('Aubrey Nelson'),
('Samuel Parker'),
('Elizabeth Hall'),
('Jack Martinez'),
('Ella Morris'),
('William Brooks'),
('Scarlett Bell'),
('David Lewis'),
('Victoria Price'),
('Henry Sanchez'),
('Luna Edwards'),
('Owen Turner'),
('Stella Rivera'),
('Daniel Cooper'),
('Nova Mitchell'),
('Sebastian Ward'),
('Zoe Murphy'),
('Lucas Martinez'),
('Penelope Walker'),
('Caleb Rogers'),
('Lily Howard'),
('Jackson Moore'),
('Sofia Perry'),
('Gabriel Morgan'),
('Avery Foster'),
('Elijah Bennett'),
('Addison Russell'),
('Grayson Diaz'),
('Hazel Johnson'),
('Ethan Myers'),
('Aria White'),
('Carter Peterson'),
('Grace Adams'),
('Jayden Hill'),
('Riley Reed'),
('Leo Turner'),
('Ella Rogers'),
('Alexander Bell'),
('Chloe Parker'),
('Henry Hayes'),
('Scarlett Wright'),
('David Cooper'),
('Layla Evans'),
('James Allen'),
('Aurora Mitchell'),
('Michael Ward'),
('Sophia Foster'),
('Jackson Reed'),
('Ava Collins'),
('William Hernandez'),
('Emily Carter'),
('Daniel King'),
('Olivia Scott'),
('Benjamin Adams'),
('Mia Stewart'),
('Noah Hughes'),
('Emma Morgan'),
('Logan Fisher'),
('Isabella Turner'),
('Liam Mitchell'),
('Avery Lopez'),
('Eleanor Howard'),
('Lucas Richardson'),
('Charlotte Miller'),
('Owen Cooper'),
('Amelia Murphy'),
('Elijah Wilson'),
('Penelope Green'),
('Caleb Martinez'),
('Lily Bailey'),
('Jack Smith'),
('Zoe Foster'),
('Ryan Johnson'),
('Aria Gray'),
('Samuel Turner'),
('Hannah White'),
('Daniel Brooks'),
('Ava Wright'),
('Matthew Parker'),
('Sophie Reed'),
('Alexander Cooper'),
('Grace Davis'),
('William Ward'),
('Mia Rogers');
```

File: database/src/init.sql (hunk 1 above)
Comment (gpseal) [reply database/src/init.sql:30]: Good point, fixed

File: database/src/init.sql (hunk 2 above)
Comment (gpseal) [reply database/src/init.sql:60]: nice idea, have implemented

**Review by AardhynLavender — APPROVED** [review]
Great, 👍.

**Pull request merged**